examples/stem-react/pages/_default/Head.tsx

```diff
@@ -8,6 +8,7 @@ const description = 'My first Vite/Stem app'
 function Head() {
   return (
     <>
```

Comment:
The `@brillout/stem-react` package's current implementation will always trigger the warning.
https://github.com/brillout/stem-react/blob/main/renderer/_default.page.server.tsx#L34